src/libraries/Common/src/System/Security/Cryptography/CompositeMLDsaManaged.ECDsa.cs

```diff
@@ -105,8 +116,28 @@ public static unsafe ECDsaComponent ImportPrivateKey(ECDsaAlgorithm algorithm, R
                             parameters.Validate();
 
                             return new ECDsaComponent(ECDsa.Create(parameters), algorithm);
-#else
-                            throw new PlatformNotSupportedException();
+#else // NETFRAMEWORK
+#if NET472_OR_GREATER
+#error ECDsa.Create(ECParameters) is avaliable in .NET Framework 4.7.2 and later, so this workaround is not needed anymore.
```

Comment:
These \#error messages are probably fine... but if we were to decide to build this library split across net462 and net472 these would just move into extra conditions on the if, and the else would survive.

They're clearly intended more for "assuming we only build one netfx variant, but we raise that number to 4.7.2, delete this code".  So, they're probably fine... I'm just saying it's not "perfect".